sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -71,6 +71,28 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
   }
 
   /**
+   * Calculates the approximate quantiles of numerical columns of a DataFrame.
```

Comment:
If we don't have the full doc from the above method, we should perhaps provide an `@see` link to the full info about the algorithm?